### ISSUE TITLE
Does not work in VS 2019 and .Net Core project

### ISSUE THREAD

**Issue by okarpov** [aberus/ImageVisualizer#10]
An unhandled exception of type 'System.Runtime.Serialization.SerializationException' was thrown by the custom visualizer component in the process being debugged.

Type 'System.Drawing.Bitmap' in Assembly 'System.Drawing.Common, Version=4.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' is not marked as serializable.

   at System.Runtime.Serialization.Formatters.Binary.WriteObjectInfo.InitSerialize(Object obj, ISurrogateSelector surrogateSelector, StreamingContext context, SerObjectInfoInit serObjectInfoInit, IFormatterConverter converter, ObjectWriter objectWriter, SerializationBinder binder)
   at System.Runtime.Serialization.Formatters.Binary.ObjectWriter.Serialize(Object graph, BinaryFormatterWriter serWriter)
   at System.Runtime.Serialization.Formatters.Binary.BinaryFormatter.Serialize(Stream serializationStream, Object graph)
   at Microsoft.VisualStudio.DebuggerVisualizers.VisualizerObjectSource.Serialize(Stream serializationStream, Object target)
   at Microsoft.VisualStudio.DebuggerVisualizers.VisualizerObjectSource.GetData(Object target, Stream outgoingData)
   at Microsoft.VisualStudio.DebuggerVisualizers.DebuggeeSide.Impl.ClrCustomVisualizerDebuggeeHost.GetData(Object visualizedObject)

**Comment by aberus** [comment]
Which version of .NET Core are you using? 2.x or 3.x+?

**Comment by okarpov** [comment]
both 5.0 and 3.0

**Comment by aberus** [comment]
I will need some help this figuring it out. System.Drawing.Bitmap is not part of .NET (Core) out of the box.

How do you reference/use System.Drawing.Bitmap in your project(s)? 
Do you use nuget packege [Microsoft.Windows.Compatibility](https://www.nuget.org/packages/Microsoft.Windows.Compatibility) or [System.Drawing.Common](https://www.nuget.org/packages/System.Drawing.Common/)? If you do, which version do you use?
Or ```<UseWindowsForms>true</UseWindowsForms>``` or something else?

In some (older) versions of this assembly, System.Drawing.Bitmap wasn't Serializable. But from .NET Core 3.0 onward it should be, so this is really strange 🤔. https://github.com/dotnet/corefx/pull/30817

**Comment by okarpov** [comment]
I use free Spire.PDF that returns System.Drawing.Image

with is located there:
#region Assembly System.Drawing.Common, Version=4.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51
// ...\.nuget\packages\system.drawing.common\4.5.0\ref\netstandard2.0\System.Drawing.Common.dll
#endregion


```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net5.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="FreeSpire.PDF" Version="8.2.0" />
    <PackageReference Include="PDFNet" Version="9.1.1.78687" />
    <PackageReference Include="PdfSharp" Version="1.50.5147" />
    <PackageReference Include="TesseractOCR" Version="5.1.0" />
  </ItemGroup>

  <ItemGroup>
    <Compile Update="Program.cs">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </Compile>
  </ItemGroup>

  <ItemGroup>
    <None Update="1.PDF">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="1.png">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="19588343bol.pdf">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="2.PDF">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="3.pdf">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="4.pdf">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="PF INVOICE TOTALS FORM .pdf">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="rd.pdf">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="tessdata\Configs\bazzar">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="tessdata\eng.traineddata">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="tessdata\eng.user-patterns">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="tessdata\eng.user-words">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="tessdata\osd.traineddata">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="tessdata\pdf.ttf">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="test_drive_1.pdf">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="x64\leptonica-1.82.0.dll">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="x64\tesseract.exe">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="x64\tesseract51.dll">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="x86\leptonica-1.82.0.dll">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="x86\tesseract.exe">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="x86\tesseract51.dll">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="_tessdata\configs\config.cfg">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="_tessdata\eng.traineddata">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="_tessdata\equ.traineddata">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="_tessdata\osd.traineddata">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="_tessdata\pdf.ttf">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
    <None Update="_tessdata\pdf.ttx">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
  </ItemGroup>

</Project>

```

**Comment by aberus** [comment]
Thank you, this was helpful, .NET Standard 2.0 suffers the same problem as .NET Core 2.x and lower version, System.Drawing.Bitmap in not marked as [Serializable].

I will publish a new version with a fix for this in the coming days.

**Comment by aberus** [comment]
New version is available in VS marketplace. Looking for feedback if it resolved this issue.

**Comment by okarpov** [comment]
does not throw an exception, but shows no image - just empty window

**Comment by aberus** [comment]
This should be fixed in 1.0.1 - will be shortly available in VS marketplace.
